config/storage.yml

```diff
@@ -0,0 +1,34 @@
+test:
```

Comment:
Do we want to force `ActiveStorage` usage instead of `Shire`? If no, could we remove this file at all?

Reply:
It depends on what we want. ActiveStorage is default rails-way gem that can be used in any project, Shrine - more powerfull and looks like NOT needed in every project.

I prefer to have default gems and default settings as more as possible. If you sure that we always must use Shrine - ok, we disable AS